roles/ceph-common/templates/ceph.conf.j2

```diff
@@ -15,7 +15,7 @@ auth service required = none
 auth client required = none
 auth supported = none
 {% endif %}
-fsid = {{ fsid }}
+{% if not mon_containerized_deployment_with_kv%}fsid = {{ fsid }}{% endif %}
```

Comment:
I've kinda missed the patch that I introduce this change. The ceph-docker project has its own mechanism to build the ceph.conf file. If you use populate_kv and generate the ceph.conf with ansible, populate becomes useless. So we should be able to allow both scenarios, either ceph-ansible generating the ceph.conf either let the container generates its own configuration file.